go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/logrusorgru/aurora v2.0.3+incompatible
 	github.com/onflow/cadence v0.42.5
 	github.com/onflow/flow-archive v1.3.4-0.20230503192214-9e81e82d4dcc
-	github.com/onflow/flow-go v0.32.4-0.20231111003039-3b479f6b0995
+	github.com/onflow/flow-go v0.32.4-0.20231115172515-c1ec969fd6f2
```

Comment:
Is this updating to a commit on `master` or is this pointing to a branch?

Reply:
its updating against a latest version of master